Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/CompositorWrapper.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// Enable this when you have access to the spectator view compositor dlls
+// #define COMPOSITOR_PLUGIN_AVAILABLE
+
+#if COMPOSITOR_PLUGIN_AVAILABLE
+using SpectatorView;
+#endif
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public class CompositorWrapper
+    {
+        public static RenderTexture GetDSLRFeed()
+        {
+#if COMPOSITOR_PLUGIN_AVAILABLE
+            // Obtain DSLR Feed
+            return ShaderManager.Instance.compositeTexture; // TODO - figure out what is the correct texture to use here in the new compositor wrapper
```

Comment:
>// TODO - figure out what is the correct texture to use here in the new compositor wrapper [](start = 60, length = 90)

Nit: If this TODO is going to be addressed very soon this is fine, if not it's gonna be a dead TODO and future folks probably won't know what to do with it. I guess either expand on it (to explain how to figure this out) or file an issue to track it.

Reply:
I"ll change it to note. This has to be changed before its used by anyone externally. The shader manager may not exist in the future.